src/ingest.py

```diff
@@ -159,7 +162,9 @@ def main():
                 logger.info(f"Ingesting {resources}")
                 print(f"*** Ingesting {resources}", file=sys.stderr)
                 try:
-                    ingester.run(resources)
+                    refs = ingester.run(resources)
+                    ids = [ref.dataId for ref in refs]
+                    visit_definer.run(ids)
```

Comment:
Maybe put the visit definition in a different try block so you can have a different log message around it?